flutter_vlc_player/README.md

```diff
@@ -146,9 +146,10 @@ class _MyHomePageState extends State<MyHomePage> {
 ```
 
 
+### Recording feature
+To start/stop video recording, you have to call the `startRecording(String saveDirectory)` and `stopRecording()` methods, respectively. By calling the stop method you can get the path of recorded file from `vlcPlayerController.value.recordPath`.
 
-
-
+<b>Note</b>: there is a bug in vlc library that if you do not call the `stopRecording()` method and the video reaches to end the recorded video path is not getting updated, because the vlc stop recording event never gets called.
```

Comment:
Let's create a `Known Issues` section closer to the bottom of the readme, and move it there.

Also, specify that:
1. The issue is specific to iOS -- I don't think I ran into the problem on Android.
2. The issue has to do with the underlying library itself, not the Dart wrapper we're supporting.

Reply:
@mitchross and I tested on a real android device and the same problem occurs. So it's not specific to IOS platform.
Since we currently support both Android & IOS platforms, i don't think we need to mention it in readme.